src/Index.php

```diff
@@ -53,14 +49,14 @@ public function getDocuments($options = null)
         return $this->httpGet('/indexes/'.$this->uid.'/documents', $options);
     }
 
-    public function addOrReplaceDocuments($documents)
+    public function addDocuments($documents, $primary_key = null)
```

Comment:
We're done with addOrReplace?

Reply:
Yes, in all the SDK we use `add_documents` and `update_documents` instead of the longer ones `add_or_replace` and `add_or_update` to be clearer. One method for one usage.